flo-runner/src/main/java/com/spotify/flo/context/ForkingContext.java

```diff
@@ -80,19 +82,25 @@ static EvalContext dryComposeWith(EvalContext baseContext) {
     LOG.debug("Test run, forking disabled - testing serialization");
```

Comment:
Let's remove this debug log because is function will be called from different paths: dry or test, where we already log.

Reply:
👍